TruckMuncher/MapViewController.swift

```diff
@@ -243,6 +265,8 @@ class MapViewController: UIViewController,
     }
     
     func updateMapWithActiveTrucks() {
+        mapView.removeAnnotations(mapView.annotations)
```

Comment:
Added this line and change the end of the method to use the mapClusterControllerSetup() method to reinitialize the cluster controller in order to fix the issue where the annotations that didn't match the search weren't being removed from the mapview